src/hotspot/share/opto/gcm.cpp

```diff
@@ -1519,6 +1532,9 @@ void PhaseCFG::schedule_late(VectorSet &visited, Node_Stack &stack) {
       // Hoist LCA above possible-defs and insert anti-dependences to
       // defs in new LCA block.
       LCA = insert_anti_dependences(LCA, self);
+      if (C->failing()) {
```

Comment:
Same here. I don't see where `record_failure()` is called from `insert_anti_dependences()`.
And there are a lot of assets there.

Reply:
This is intended to cover `insert_anti_dependences` : `memory_early_block` : `assert_dom` : `record_failure("unschedulable graph");`